src/Services/Export/WB.Services.Export/InterviewDataStorage/Services/QuestionnaireSchemaGenerator.cs

```diff
@@ -206,15 +206,23 @@ private string GetSqlTypeForVariable(Variable variable)
                     throw new ArgumentException("Unknown variable type: " + variable.Type);
             }
         }
+        
+        private object lockObj = new object();

```

Comment:
Two requests will create two instances. No value of it here.

Reply:
removed